docs/Software & Tools/ColabFold.md

```diff
@@ -0,0 +1,259 @@
+
+## Getting started
+### Search against the ColabFold databases
+
+Finding homologous proteins using MMSeqs2 can be done by running a batch script using the SBATCH command: `sbatch <path to batch script file>`
+
+The code below is an example of a batch script to run MMSeqs2. The top of the script contains the instruction to use bash to execute the commands (`#!/bin/bash`) and the SBATCH parameters (`#SBATCH <parameter>`) followed by the different modules required to run MMSeqs2 through ColabFold.
+
+
+```
+#!/bin/bash
+#SBATCH --partition=cpu
+#SBATCH --nodes=1
+#SBATCH --cpus-per-task=32
+#SBATCH --mem=200G
+#SBATCH -t 05:00:00
+#SBATCH -o slurm-%j.out
+#SBATCH -e slurm-%j.err
+
+module load uri
+module load MMseqs2/14-7e284-gompi-2021b
+module load miniconda
+source ~/.bashrc
+conda activate colabfold
+```
+
+
+The command colabfold_search shown below is added after activating the conda environment `colabfold-v1.5.0`. In this case, protein sequences contained in a fasta file are aligned against the UniRef30 (`--db1 uniref30_2202/uniref30_2202_db`) and environmental (`--db3 colabfold_envdb_202108/colabfold_envdb_202108_db`) databases. UniRef30 is the default database used to search proteins. In order to use the environmental database, the parameter `--use-env `has to be set to 1 in addition to providing the path (`--db3 colabfold_envdb_202108/colabfold_envdb_202108_db`).
+
+
+```
+colabfold_search <path to fasta file> /datasets/bio/colabfold <path to output directory> --db1 uniref30_2202/uniref30_2202_db --db3 colabfold_envdb_202108/colabfold_envdb_202108_db --use-env 1 --use-templates 0
+```
+
+
+To use the PDB70 templates database, the parameter `--use-templates` should be set to 1 and the path to the database should be provided with `--db2 pdb`.
+
+#### Notes: 
+* It is recommended to request at least 200G using `#SBATCH --mem=200G` in order to load the protein databases.
+* Running colabfold_search with 1,762 proteins, the UniRef30 and environmental databases and the highest mmseqs sensitivity (s = 8) on a gpu A100 node with 64 threads takes approximately 3h.
+
+### Train models and make predictions
+
+The code below provides an example on how to make predictions using `colabfold_batch` in a batch script:
+
+The parameter --stop-at-score is used to stop generating models until the predicted confidence metric (pLDDT or predicted local distance difference test) is reached.
+
+
+```
+#!/bin/bash
+#SBATCH --partition=uri-gpu
+#SBATCH --nodes=1
+#SBATCH --cpus-per-gpu=16
+#SBATCH -mem-per-gpu=80
+#SBATCH -t 05:00:00
+#SBATCH -o slurm-%j.out
+#SBATCH -e slurm-%j.err
+
+module load cudnn/cuda11-8.4.1.50
+module load cuda/11.4.0
+module load miniconda
+source ~/.bashrc
+
+conda activate colabfold
+
+colabfold_batch <path to directory containing MSAs> <path to output directory> --stop-at-score 85 --msa-mode 'MMseqs2 (UniRef+Environmental)'
```

Comment:
if these paths need to be specified explicitly, maybe include something about where they might be found.